src/rules.js

```diff
@@ -1903,6 +1903,80 @@ module.exports = [
     },
     /**
      ==================================================================
+     TRANSITION
+     ==================================================================
+     */
+    {
+        type: 'pattern',
+        id: 'transition',
+        name: 'Transition',
+        prefix: '.Trs-',
+        properties: ['transition'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true
+    },
+    {
+        type: 'pattern',
+        id: 'transition-delay',
+        name: 'Transition delay',
+        prefix: '.Trsde-',
+        properties: ['transition-delay'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'i', values: ['initial']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-duration',
+        name: 'Transition duration',
+        prefix: '.Trsdu-',
+        properties: ['transition-duration'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
+        rules: [
+            {suffix: 'inh', values: ['inherit']}
+        ]
+    },
+    {
+        type: 'pattern',
+        id: 'transition-property',
+        name: 'Transition property',
+        prefix: '.Trsp-',
+        properties: ['transition-property'],
+        allowCustom: true,
+        allowCustomAutoSuffix: false,
+        allowSuffixToValue: true,
```

Comment:
I think there is value in allowing people to do this because in any case *there is good chance that's the string they'd be using* to create their custom value. 
And we could namespace global suffixes to avoid any confusion, no?